cniconf/watcher.go

```diff
@@ -78,7 +83,39 @@ func (w *watcher) onChange(version string) error {
 	return nil
 }
 
-func (w *watcher) apply(network metadata.Network) error {
+func (w *watcher) inserthostSubnetRules(host metadata.Host, content []byte) ([]byte, error) {
+	subnet, _ := host.Labels[hostSubnetLabel]
```

Comment:
@leodotcloud Where do I set the label? If it is set on the CNI container, it cannot correspond to the managed network. Because the CNI container uses net=host.